scripts/call_variants.md

```diff
@@ -0,0 +1,35 @@
+
+# call_variants.py
+
+## Help message
+```
+Description: The program calls SNPs and InDels from the unique isoform file
+
+positional arguments:
+  inFile      string	input file.  e.g. /in/unique_isoforms.tsv
+  outFile     string	output file.  e.g. /out/unique_isoforms_variants.tsv
+
+optional arguments:
+  -h, --help  show this help message and exit
+  -n , --N    float	maximum bases for indel combination (inclusive), default 10
+```
+
+## Running script
+```
+python3 call_variants.py unique_isoforms_and_aln_stats_filter0.99.tsv unique_isoforms_and_aln_stats_filter0.99.call.tsv --N 10
+```
+
+## Output
+ref_id: reference sequence ID
+query_id: query sequence ID
+type: type of variant (snp/ins/del/delins)
+ref: variant sequence in reference
+alt: altered sequence
+ref_start: starting position of the variant in reference
+```
+#ref_id	query_id	type	ref	alt	ref_pos
```

Comment:
I would put these in the standard VCF format:

```
CHR   REF_POS   REF ALT   TYPE   REF_ID.   QUERY_ID  
```

Please note in the comments that the position should be 1-based, which is standard for VCF (and please doublecheck that this is the case)

@BidaGu

Reply:
Spoke with Bida about this just now. 

We'll update a markdown file to record the file format here.

Reply:
Roughly, here's the summary:

We do a pairwise alignment between the REFERENCE and the QUERY.

* chromosome on reference/assembly
*starting position of REFERENCE on reference/assembly
* pos of variant on pairwise alignment
* Pair of IDs used for REF and QUERY, i.e. some sort of IDs for the pairwise alignment
* REF "allele" (whatever the bases are here; this is standard for VCF)
* ALT "allele" (whatever the bases are here; this is standard for VCF)
* the REFERENCE sequence
* the QUERY sequence

We should try to copy the VCF roughly, as this is what users will expect:

```
CHROM   POS   ID   REF   ALT 
```

https://en.wikipedia.org/wiki/Variant_Call_Format

Reply:
Here's a PR documenting this file format: https://github.com/collaborativebioinformatics/isocomp/pull/22